launch_snakemake.py

```diff
@@ -260,7 +260,7 @@ def analysis_main(args, runnormal, runtumor, output, normalname, normalfastqs, t
     if os.path.isfile(f"{output}/reporting/workflow_finished.txt"):
         # these functions are only executed if snakemake workflow has finished successfully
         yearly_stats(args.tumorsample, args.normalsample)
-       # petagene_compress_bam(args.outputdir, args.tumorsample)
+        petagene_compress_bam(args.outputdir, args.tumorsample)
```

Comment:
You could give it a go and squash this commit with the previous commit (as they cancel each other out). This would remove them both so you've got a cleaner commit history.